es-app/src/CollectionSystemManager.cpp

```diff
@@ -427,90 +400,7 @@ void CollectionSystemManager::exitEditMode()
 	mEditingCollection = "Favorites";
 }
 
-// adds or removes a game from a specific collection
-bool CollectionSystemManager::toggleGameInCollection(FileData* file)
```

Comment:
Again, something to be implemented by the subclass itself, rather than to have the CollectionManager take responsibility.

Reply:
No objection.